src/index.js

```diff
@@ -0,0 +1,60 @@
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import {
+  getLastPackageVersion,
+  getPackageVersion,
+  setPackageVersion,
+} from './local-settings'
+import initializeSentryIfEnabled from './sentry'
+
+// Initialize Sentry as early as possible, if enabled
+initializeSentryIfEnabled()
+
+const packageVersion = getPackageVersion()
+const lastPackageVersion = getLastPackageVersion()
+
+const [currentMajorVersion, currentMinorVersion] = packageVersion.split('.')
+const [lastMajorVersion, lastMinorVersion] = lastPackageVersion.split('.')
+
+// Purge localstorage when upgrading between different minor versions.
+if (
+  lastMajorVersion !== currentMajorVersion ||
+  lastMinorVersion !== currentMinorVersion
+) {
+  window.localStorage.clear()
+
+  // Attempt to clean up indexedDB storage as well.
+  if (
+    window.indexedDB &&
+    window.indexedDB.databases &&
+    window.indexedDB.deleteDatabase
+  ) {
+    // eslint-disable-next-line promise/catch-or-return
+    window.indexedDB
+      .databases()
+      .then(databases =>
+        databases.forEach(({ name }) => window.indexedDB.deleteDatabase(name))
+      )
+  }
+}
```

Comment:
I only kept this for the sentry config, do we really need it right now? is there a simpler alternative in this new world?

Reply:
No this was to clear the aragon.js cache, we can remove it here :+1: